VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java

```diff
@@ -71,4 +80,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		}
 		return false;
 	}
+
+  @Override
+  public void update(Observable o, Object arg) {
+    super.update(o, arg);
+    changeToolbarVisibilityInFullscreenMode(arg);
+  }
+
+  private void changeToolbarVisibilityInFullscreenMode(Object arg) {
```

Comment:
Is this problem appear only for our sample app?

Reply:
They don't use toolbar